DayrSuryan/DSEthiop8.xml

```diff
@@ -643,11 +643,23 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                      </list>
                   </additions>
 
+                  <bindingDesc>
+                     <binding xml:id="binding">
+                        <decoNote xml:id="b1">Egyptian binding.
+                          Cardboard, reddish leather. Endbands.</decoNote>
+                        <decoNote xml:id="b2" type="bindingMaterial">
+                           <material key="leather"/>                          
+                           <material key="textile"/>
+                        </decoNote>
+                        <decoNote xml:id="b3" type="SewingStations"/>
```

Comment:
The note with sewingStations is empty

Reply:
I did; I now see that I am not very sure about this feature.